iOS/DuckDuckGo/SERP/SERPSettingsProviding.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  SERPSettingsProviding.swift
+//  DuckDuckGo
+//
+//  Copyright © 2025 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+public protocol SERPSettingsProviding {
+    var isDuckAIEnabled: Bool { get }
```

Comment:
The project uses "AIChat" to reference duck.ai, since duck.ai is the marketing name and it can change like it changed before. Would be good to keep using AIChat when possible.